v22.1/create-changefeed.md

```diff
@@ -143,7 +143,7 @@ Option | Value | Description
 <a name="initial-scan"></a>`initial_scan` | `yes`/`no`/`only` |  Control whether or not an initial scan will occur at the start time of a changefeed. Only one `initial_scan` option (`yes`, `no`, or `only`) can be used. If none of these are set, an initial scan will occur if there is no [`cursor`](#cursor-option), and will not occur if there is one. This preserves the behavior from previous releases. With `initial_scan = 'only'` set, the changefeed job will end with a successful status (`succeeded`) after the initial scan completes. You cannot specify `yes`, `no`, `only` simultaneously. <br><br>If used in conjunction with `cursor`, an initial scan will be performed at the cursor timestamp. If no `cursor` is specified, the initial scan is performed at `now()`. <br><br>Although the [`initial_scan` / `no_initial_scan`](../v21.2/create-changefeed.html#initial-scan) syntax from previous versions is still supported, you cannot combine the previous and current syntax. <br><br>**Note**: You cannot use the new `initial_scan = "yes"/"no"/"only"` syntax with [`ALTER CHANGEFEED`](alter-changefeed.html) in v22.1. To ensure that you can modify a changefeed with the `initial_scan` options, use the previous syntax of `initial_scan`, `no_initial_scan`, and `initial_scan_only`.<br><br>Default: `initial_scan = 'yes'`
 `kafka_sink_config` | [`STRING`](string.html) |  Set fields to configure the required level of message acknowledgement from the Kafka server, the version of the server, and batching parameters for Kafka sinks. See [Kafka sink configuration](changefeed-sinks.html#kafka-sink-configuration) for more detail on configuring all the available fields for this option. <br><br>Example: `CREATE CHANGEFEED FOR table INTO 'kafka://localhost:9092' WITH kafka_sink_config='{"Flush": {"MaxMessages": 1, "Frequency": "1s"}, "RequiredAcks": "ONE"}'`
-<a name="full-table-option"></a>`full_table_name` | N/A | Use fully-qualified table name in topics, subjects, schemas, and record output instead of the default table name. This can prevent unintended behavior when the same table name is present in multiple databases. <br><br>Example: `CREATE CHANGEFEED FOR foo... WITH full_table_name` will create the topic name `defaultdb.public.foo` instead of `foo`.
+<a name="full-table-option"></a>`full_table_name` | N/A | Use fully-qualified table name in topics, subjects, schemas, and record output instead of the default table name. This can prevent unintended behavior when the same table name is present in multiple databases.<br><br>**Note:** This option cannot modify existing table names used as topic names etc. as part of an [`ALTER CHANGEFEED`](alter-changefeed.html) statement. To modify a topic name, output etc., create a new changefeed with this option. <br><br>Example: `CREATE CHANGEFEED FOR foo... WITH full_table_name` will create the topic name `defaultdb.public.foo` instead of `foo`.
```

Comment:
supernit: I would just match the truncations `topic names etc.` and `topic name, output etc.` to be the same. Maybe:

> **Note:** This option cannot modify existing table names used in topics, subjects, etc. as part of an [`ALTER CHANGEFEED`](alter-changefeed.html) statement. To modify a topics, subject, etc., create a new changefeed with this option.